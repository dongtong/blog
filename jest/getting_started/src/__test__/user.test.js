import User from '../user';

describe("User Test Suite", () => {
  test("Get full name", () => {
    const user = new User({firstName: 'foo', lastName: 'bar'});
    expect(user.fullName).toBe('foo bar');
  });
});