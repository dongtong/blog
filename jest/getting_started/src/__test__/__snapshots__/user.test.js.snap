// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Hooks test last user snapshot 1`] = `
User {
  "firstName": "foo",
  "lastName": "bar",
}
`;

exports[`User snapshot test user snapshot 1`] = `
User {
  "firstName": "foo",
  "lastName": "bar2",
}
`;
